app/Http/Controllers/Api/ApiLinksController.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace App\Http\Controllers\Api;
+
+use Illuminate\Http\Request;
+
+use App\Models\Link;
+
+class ApiLinksController extends ApiController
```

Comment:
Is it necessary to create a new class for these two functions? If so, perhaps we can name them more distinctly so that they don't get confused. Perhaps `ApiLinkEditController`?